src/apps/opening-hours-editor/OpeningHoursEditor.entry.tsx

```diff
@@ -36,8 +37,13 @@ const OpeningHoursEditorEntry: React.FC<
   OpeningHoursEditorEntryTextProps &
     OpeningHoursEditorType &
     OpeningHoursEditorEntryConfigProps
-> = ({ initialDate = new Date() }) => {
-  return <OpeningHoursEditor initialDate={initialDate} />;
+> = ({ initialDate }) => {
+  const initialDateParam = getInitialDateFromUrl();
+  return (
+    <OpeningHoursEditor
+      initialDate={initialDate ?? (initialDateParam || undefined)}
```

Comment:
I’m considering whether `initialDate={initialDate ?? (initialDateParam || new Date())}` should be here. However, `OpeningHoursEditor` operates as expected without an `initialDate`, so I’d rather avoid passing a `new Date()`.

Reply:
I wonder why you want to avoid passing `new Date()`. As I see it this was what we did before if `initialDate` was not passed.